src/components/custom/_dhero.tsx

```diff
@@ -52,6 +52,10 @@ const ImageWrapper = styled.div`
     @media ${device.tablet} {
         padding: 64px 0 54px;
     }
+
+    & > a {
+        z-index: 1;
+    }
```

Comment:
Hi @hasan-deriv just a question, Why do we need the `z-index` here for the `<a>` tag 🙏 ?

Reply:
The background shape was moving over the image